File: src/API/WesternStatesWater.WestDaat.Contracts.Client/AnalyticsSummaryInformationResponse.cs
namespace WesternStatesWater.WestDaat.Contracts.Client;

public class AnalyticsSummaryInformationResponse
{
    public AnalyticsSummaryInformation[] AnalyticsSummaryInformation { get; set; }
    public GroupItem[] GroupItems { get; set; }
    public int GroupValue { get; set; }
}

File: src/DashboardUI/src/data-contracts/AnalyticsSummaryInformationResponse.ts
import { AnalyticsSummaryInformation } from './AnalyticsSummaryInformation';
import { GroupItem } from './GroupItem';

export interface AnalyticsSummaryInformationResponse {
  analyticsSummaryInformation: AnalyticsSummaryInformation[];
  groupItems: GroupItem[];
  groupValue: number;
}
